changelog.yaml

```diff
@@ -16,7 +16,24 @@
 
 - version: "NEXT"
   date: TBD
-  
+  changes:
+  - type: enhancement
+    impact: minor
+    title: Make Jenkinsfile Runner properties configurable in PipelineRun custom resource objects.
+
+    description: >-
+      Jenkins filerunner properties are configurable in Steward-PipelineRun manifest at 'spec.jenkinsfileRunner' now.
+    warning: ""
+    upgradeNotes: >-
+      No incompatible changes.
+      Deprecation Notice:

```

Comment:
Sounds good. Better than `upgradeNotes`. But it has to be clear if the deprecation causes an incompatibility or if it can be consumed without any adaptations.

Reply:
Like this? (first in release concept template)
https://github.wdf.sap.corp/ConCISe/OperationsGuide/pull/355/files

Reply:
I would keep upgradeNotes for things which MUST be considered.
Deprecation for me is just an announcement that something will be changed in the future.